PD-1667 / 25.04 / PD-1667-investigate-and-document-disk-replacement-procedure-for-non-hotswappable-systems (#3652)
truenas/documentation: content/SCALETutorials/Storage/Disks/ReplacingDisks.md

After offlining the failed disk, physically remove it from the system.

### Replacing a Failed Disk in a Non-Hotswappable System

> To replace a failed disk in a non-hotswappable system, first, identify the serial number of the failed disk. This information can be located in the **Disks** section of the **Storage** screen and is crucial for ensuring that you finish the replacement process on the correct disk.

Reviewer: tense and voice suggestion,
```suggestion
To replace a failed disk in a non-hotswappable system, first, identify the serial number of the failed disk. To find the serial number, click on **Disks** on the **Storage** screen. The serial number is crucial for ensuring that you finish the replacement process on the correct disk.
```

> After you have identified the failed disk, [take the failed disk offline](#taking-a-failed-disk-offline). Then, completely shut down your system. This is a vital step for non-hotswappable systems, as disks cannot be removed or inserted in a non-hotswappable system if the power is still on.

Reviewer: voice and tense suggestion, and it doesn't matter if the disk is hot-swappable, you have to shut down the system for any disk replacement.
```suggestion
After identifying the failed disk, [take the failed disk offline](#taking-a-failed-disk-offline). Then, completely shut down your system. You cannot remove or insert disks if the power is still on.
```

Reviewer: You are saying they can be in your comment but the test says it cannot be. Which is it?

Author: No, I'm saying in my proposed change that if your system is NOT hotswappable, you have to power it off first. In my comment I'm saying that if your system IS hotswappable, you don't have to power it off first--which is why I'm emphasizing the importance of powering off your NOT hotswappable systems in my original proposed change.

> When you have ensured that your system is powered off, remove the failed drive from the system. Double-check the serial number on the drive you are removing to verify that it is the failed drive you previously identified.

Reviewer: ```suggestion
After your system is powered off, remove the failed drive from the system. Double-check the serial number on the removed drive matches the failed drive you previously identified.
```

> Connect the replacement disk and reboot your system. Depending on your hot spare availability, continue with one of the below processes to complete the resilver process.

Reviewer: for clarity
```suggestion
Insert and connect the replacement disk and power-up your system. Complete the resilver process below that matches your hot spare availability.
```
